docs/get-started/Join-the-Mainnet.md

```diff
@@ -0,0 +1,44 @@
+# IRIShub Network
+
+
+## How to Join IRIShub Mainnet
+
+### Step 1: Install Iris on Your Server
+
+Please follow this [instruction](Install-the-Software.md) to get **Iris** installed locally.
+
+### Step 2: Run a Full Node
+
+Please follow this [instruction](Full-Node.md) to get your full node running.
+
+
+### Step 3: Upgrade to Validator Node
+
+Please follow this [instruction](Validator-Node.md) to upgrade your full node to validator node.
+
+### Deploy IRISHub Monitor
+
+Please follow this [guide](../software/monitor.md) to get IRIHub monitor running on your side.
+
+
+### Upgrade to Validator Node
+
+You now have an active full node. What's the next step? 
+
+If you have participated in the genesis file generation process, you should be a validator once you are fully synced. 
+
+If you miss the genesis file generation process, you can still upgrade your full node to become a IRISnet Validator. The top 100 validators have the ability to propose new blocks to the IRIS Hub. Continue onto [the Validator Setup](Validator-Node.md).
+
+### Setup a sentry node
+
+A validator is under the risk of being attacked. You could follow this [guide](../software/sentry.md) to setup a sentry node to protect yourself.
+
+## Fuxi  Testnet
+
+Fuxi testnet is used for testing irishub.
```

Comment:
The Fuxi testnet is used to provide a stable test environment for developers such as third-party wallets, browsers and other applications.